tests/usecase/test_emit_metrics.py

```diff
@@ -15,6 +21,29 @@ def test_read_metrics_in_logs(self):
         self.assert_string_in_recent_logs("number_of_files")
         self.assert_string_in_recent_logs("critical_logs_count")
 
+    def test_free_apps_metrics(self):
+        self.setUpCF(
+            "sample-6.2.0.mda",
+            env_vars={"METRICS_INTERVAL": "10", "PROFILE": "free"},
+        )
+        self.startApp()
+
+        time.sleep(10)
+        self.assert_string_in_recent_logs("MENDIX-METRICS: ")
+        self.assert_string_in_recent_logs("named_users")
+        self.assert_string_in_recent_logs("anonymous_sessions")
+        self.assert_string_in_recent_logs("named_user_sessions")
+
+
+mock_user_session_metrics = {
```

Comment:
`mock_user_session_metrics` is defined on line 38, but the class that uses it is defined on line 111.

Can we a) move it closer to the class that uses it and b) consider making it a member variable of the class that uses it (only one class uses it)